### DISCUSSION TITLE
test_client cannot connect with nginx-quic becauseof validate token failed

### DISCUSSION THREAD

**Discussion by innocenceman** [alibaba/xquic#87] [Q&A] [answered]
What happened?
xquic client connect to **nginx-quic** failed, and there is a error "**_quic garbage token while handling initial packet_**" in error.log
xquic was built with boringssl, and the client is test_client, client show "conn errno:11" in the end.

Steps To Reproduce
download nginx-quic with version [release-1.21.6](https://hg.nginx.org/nginx-quic/rev/release-1.21.6), build and install it, run nginx
with nginx.conf:"listen 8443 http3 reuseport". 
use test_client(command line: ./test_client -a xxx.xxx.xxx.xxx -p xxxx -G -l e -u https://www.xxxxx.com/) to request nginx-quic server.

thanks
xquic.zip

**Comment by ruiqizhou** [comment, marked as answer]
#92
